proposals/open-source-programs/PROPOSAL_001.md

```diff
@@ -0,0 +1,180 @@
+            * Issuing a Credential
+        * Contains:
+            * Tutorials
+            * Sample Applications
+            * Playground (if appropriate)
+            * Projects used (links to Projects pages)
+            * ...others from "Documentation Content Types"
+* Projects
+    * Open format in MDX or Markdown for Project Leads to surface project information on the Developer Site.
+    * One per top-level project, for example:
+        * SSI
+        * DWN
+        * Wallet
+        * tbDEX
+        * Web5
+    * Shows the components which make up a Project
+        * Example for SSI:
+            * ssi-service
+            * ssi-sdk
+        * Example for Web5:
+            * Web5 API
+* Documentation Content Types
+    * Additionally let the developer explore by Documentation Type as a list, rather than only by use case **Guides**?
+        * **Tutorials**
+        * **Sample Applications**
+        * **Playgrounds**
+
+## Proposed Work items to break off into Issues
+* Information Architecture - how is information organized and presented?
+* UX and Wireframe
+    * From Bobbilee's base work in Figma. Incorporates decisions made in the "Information Architecture" task
+    * Evolve Figma doc to incoporate decisions we make here. Practically-speaking, how is that done?
+    * Potential handoff point to high-fidelity design phase
+* Site Development
+    * Inventory of new components that need to be created
+    * Moving from pure Markdown publishing to MDX
+    * Guide to content authors documenting the above - MDX structure, components available, and how to use the components.
+* Tutorial structure
+    * An author's guide for writing tutorials
+    * Is there a common format all tutorials take?
+    * Includes:
+        * Best practices?
+        * Template?
+* Sample Applications structure
+    * An author's guide for writing sample applications
+    * Is there a common format all sample applications take?
+* Content authoring
+* Phase 3 Backlog:
+    * Design tasks from UX wireframes
+    * Web development to implement Designs
+
+## Implementation Considerations
+* Embed into every Project section a link to contribute - and make sure it's offshoring into appropriate homes
+* Link out to FAQs and Glossary on the Dev Site where appropriate (and/or tooltip)
+* Embed feedback forms where appropriate
+* Pop outs to choose software version where tutorials are, or where code is embedded
+* Embed media into guides where appropriate, perhaps cross-linking into where they're "housed" in Media section. Perhaps the "embed" is exandable as an option and doesn't take up lots of real estate unless user opts into.
+
+## External Examples
+* [Use Case Approach from Mattr](https://learn.mattr.global/docs/)
+* From @dcrusso: [leverage GitHub Pages to do really simple playground](https://github.com/TBD54566975/web5-components/blob/main/.github/workflows/Pages.yml) NOTE: @bobbilee19 suggests calling this "sandbox". Align on language?
```

Comment:
s/dcrusso/dcrousso :(

Reply:
Fixed in 8933d150c3bfdb4fca4017b0fb8148399fa4b9b4 with my apologies @dcrousso!